### PR TITLE
7suyash7 notes app

### PR DESCRIPTION
**Pull request by 7suyash7**
## Description
A Note App which allows you to save your Notes and even search for them later on!

## Add GIF/ScreenShot 

noteapp

## Author Name
[Suyash Nayan ](https://github.com/7suyash7)

## Task Checklist
Put x in between the square brackets without any spaces to tick your Task.  Create a PR only after you complete all the tasks.
- [x] You have created a new branch for contribution.
- [x] Added README.md for your project in a specified folder.
- [x] Add Demo in GIF or Screenshot form.
- [x] You are Registered Officially for HacktoberFest 2021 as Partcipant.
- [x] Followed Contribution Guidelines.

### CODE REVIEW

**Comment by Harshil-Jani** [comment]
@7suyash7 
Update your Readme file 
- Project Name
- Short Description
- .gif of working project.
- Author name 

And we are good to go ! :star2:

**Review by soumik2012 — CHANGES_REQUESTED** [review]
Readme.md File should be contained 
- Name of Project
- Demo of Project in .gif how your project is look 
- Description of project 
-  Requirements or Setup instructions
-  Name of Author(s)  
- [Look Here](https://github.com/Tenet-Coding/Hacktoberfest-Projects/blob/master/ConwaysGameOfLife/README.md)

**Comment by 7suyash7** [comment]
readme.md has been updated @Harshil-Jani please check it once!

**Comment by Harshil-Jani** [comment]
@7suyash7 You have deleted your Readme file ! We can't see it. Add it again so that we can merge your work.

**Comment by 7suyash7** [comment]
added the readme file again pls check.

File: NotesApp(7suyash7)/index.html [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+                    crossorigin="anonymous">
+                </script>
+                <!-- <script src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.9.3/dist/umd/popper.min.js"
+                    integrity="sha384-W8fXfP3gkOKtndU4JGtKDvXbO53Wy8SZCQHczT5FMiiqmQfUpWbYdTil/SxwZgAN"
+                    crossorigin="anonymous"> -->
+                </script>
+                <script src="script.js"></script>
+</body>
+
+</html>
```
Comment (Harshil-Jani) [inline NotesApp(7suyash7)/index.html:79]: New line at the end of file.

File: NotesApp(7suyash7)/script.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    Array.from(noteCards).forEach(function(element){
+        let cardTxt = element.getElementsByTagName("p")[0].innerText;
+        if(cardTxt.includes(inputVal)){
+            element.style.display = "block";
+        }
+        else{
+            element.style.display = "none";
+        }
+    })
+})
```
Comment (Harshil-Jani) [inline NotesApp(7suyash7)/script.js:73]: New line at the end of file

**Review by Harshil-Jani — CHANGES_REQUESTED** [review]

**Review by soumik2012 — CHANGES_REQUESTED** [review]
Make the changes then we are good to go

**Review by Harshil-Jani — APPROVED** [review]

**Pull request merged**